drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/analysis/cache/RuleInspector.java

```diff
@@ -56,6 +71,44 @@ public RuleInspector( final Rule rule,
 
             patternInspectorList.add( patternInspector );
         }
+
+        updateBRLActionInspectors( rule.getActions()
+                                       .where( Action.superType().is( ActionSuperType.BRL_ACTION ) )
+                                       .select().all() );
+        rule.getActions()
+            .where( Action.superType().is( ActionSuperType.BRL_ACTION ) )
+            .listen().all( new AllListener<Action>() {
```

Comment:
FYI, this type of construct can be simplified with lamdas to read ```.listen().all( (all)->updateBRLActionInspectors(all) );``` or even (IIRC) ```.listen().all( (all)->this::updateBRLActionInspectors );``` (which IMO is slightly *less* readable!)